src/content/reference/kernel-settings/_index.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Tuned kernel settings
+description: Complete list of the tuned CoreOS kernel settings for the Giant Swarm clusters.
+date: 2020-02-20
+layout: subsection
+weight: 100
+---
+
+# Kernel settings
+
+For the Giant Swarm tenant clusters and the applications running in it to run properly, some kernel settings need to be tuned at the OS level.
+
+## General Performance and Security settings
+| Setting                                | Value         | Description                                                 |
+| ---                                    | ---           | ---                                                         |
+| kernel.kptr_restrict    	             | 2             | Hide kernel pointers to mitigate the kernels attack surface |
+| kernel.sysrq   	                     | 0             | Reduce the kernel's attack surface                          | 
+| net.ipv4.conf.all.log_martians  	     | 1             |                                                             |  
```

Comment:
What about adding a description line for all settings?

Reply:
Let me get one